lib/recipe/bloc/recipe_bloc.dart

```diff
@@ -825,4 +853,163 @@ class RecipeBloc extends Bloc<RecipeEvent, RecipeState> {
       }
     }
   }
+
+  Recipe? _recipeOnDevice(recipeNumber) {
+    List<Recipe> deviceRecipes = List<Recipe>.from(state.recipes);
+    for (Recipe recipe in deviceRecipes) {
+      if (recipe.recipeNumber == recipeNumber) {
+        return recipe;
+      }
+    }
+    return null;
+  }
+
+  Future<void> _onRecipeImported(
+      RecipeImported event, Emitter<RecipeState> emit) async {
+    emit(state.copyWith(recipeImportStatus: '')); // Clear status for snackbar
+    FilePickerResult? result = await FilePicker.platform.pickFiles(
+        // TODO: Implement extension restriction once csv is supported by package
+        // type: FileType.custom,
+        // allowedExtensions: ['csv'],
+        );
+
+    if (result != null) {
+      final input = File(result.files.single.path!).openRead();
+      List data = await input
+          .transform(utf8.decoder)
+          .transform(const CsvToListConverter())
+          .toList();
+      data.removeAt(0); // Remove headings line in csv
```

Comment:
I think the comment could be removed and have a constant `headingsLinePosition = 0` declared and then used here `data.removeAt(headingsLinePosition);`